src/finfield_conway.h

```diff
@@ -0,0 +1,6957 @@
+/**************************************************
+ * Table of Conway polynomials for fields up order
+ * 2^32. The polynomial for q = p^r 
+ * will be of the form x^r + \sum{a_i x^i} is represented
+ * by two entries in PolsFF: q and \sum{Int(a_i) q^i}
```

Comment:
I assume this was generated by a script? Can we include that somewhere?

Reply:
I don't have it, although it would be easy to write such a script in GAP.

Reply:
I used indeed a simple GAP script. This is something I played with about two years ago, now it seems to give errors under gap4r10. But here is the code I used:
ConwayPolNumber := function(p,h)
local list,number;
list := ConwayPol(p,h);
number := Sum(List([1..Length(list)-1],i->list[i]*p^(i-1)));
end;

ConwayPolForCCodeFlat := function(p,h)
local list, number;
list := ConwayPol(p,h);
number := Sum(List([1..Length(list)-1],i->list[i]*p^(i-1)));
return Concatenation(String(p^h),", ",String(number),",\n");
end;


ConwayPolForCCodeNice := function(p,h)
local list,substring,co,sub,i,lc,lpp,empty,idx,a,stop;
empty := Concatenation(List([1..40],x->" "));
list := ConwayPol(p,h);
substring := [String(p^h)];
Add(substring,", ");
for i in [1..Length(list)-1] do
    co := list[i];
    if co = 0 then
        lpp := Length(String(p^(i-1)));
        sub := empty{[1..2+lpp]};
    else
        if i = 1 then
            sub := Concatenation(String(co)," ");
        else
            sub := Concatenation("+",String(co),"*",String(p^(i-1)));
        fi;
    fi;
    Add(substring,sub);
od;
idx := Length(substring);
stop := false;
while not stop do
    a := substring[idx];
    NormalizeWhitespace(a);
    if a = "" then
        Unbind(substring[idx]);
        idx := idx - 1;
    else
        stop := true;
    fi;
od;
Add(substring,",\n");
return Concatenation(substring);
end;

MAX := 2^32;

polynomials_as_strings_of_numbers := [];
p := 2;
h := 2;
while p^h <= MAX do
    while p^h <= MAX do
        Add(polynomials_as_strings_of_numbers,ConwayPolForCCodeNice(p,h));
        h := h+1;
    od;
    h := 2;
    p := NextPrimeInt(p);
od;

polynomials_as_strings_of_flat_numbers := [];
p := 2;
h := 2;
while p^h <= MAX do
    while p^h <= MAX do
        Add(polynomials_as_strings_of_flat_numbers,ConwayPolForCCodeFlat(p,h));
        h := h+1;
    od;
    h := 2;
    p := NextPrimeInt(p);
od;


SizeScreen([256,40]);
LogTo("~/output");
for p in polynomials_as_strings_of_numbers do
Print(p);
od;

SizeScreen([256,40]);
LogTo("~/output2");
for p in polynomials_as_strings_of_flat_numbers do
Print(p);
od;

Reply:
I added a tweaked version of this as `etc/gen_conway.g`. TODO: add a comment to src/finfield_conway.h which mentions etc/gen_conway.g